code/modules/mob/living/simple_animal/corpse.dm

```diff
@@ -251,3 +251,33 @@
 	id_job = "SolGov Sonnensoldner"
 	outfit = /datum/outfit/job/solgov/sonnensoldner
 	id_access_list = list(ACCESS_SOLGOV)
+
+//inteq
+
+/obj/effect/mob_spawn/human/corpse/inteq
```

Comment:
I don't really want stocked corpses in a ruin like this.

Reply:
what would you suggest trimming it down to?